access-portal/backend/donationNotes.inc

```diff
@@ -35,6 +41,13 @@ if ($val = $row['donor_retrospective']) {
   array_push($notes_pre_key_val_pairs, sprintf("<strong>Donor retrospective of the donation:</strong> %s", trim(cleanNotes($val))));
 }
 $notes_key_val_pairs = array();
+if ($row['amount_original_currency']) {
```

Comment:
Can we filter out the case where donor_amount_reason exists?

Reply:
Did this now https://github.com/vipulnaik/donations/pull/135/commits/ecfbdd2672517edc65280b9fed3af495908599b6